### ISSUE TITLE
http image links in trivia

### ISSUE THREAD

**Issue by petzku** [Cog-Creators/Red-DiscordBot#944]
Some trivia question sets (such as dota2abilities and dota2items) use http to link to images, not https. This means embeds aren't shown in Discord.

Should be easy to fix by just bulk replacing `http(?!s)` with `https`.

**Comment by petzku** [comment]
If trivia questions are going to be just removed from the repo (as suggested in #917), this can maybe be ignored.

**Comment by palmtree5** [comment]
Just checked, with one image selected from one of those lists, the image still doesn't show up even if one uses https instead of http

**Comment by palmtree5** [comment]
Another thing: The cars list works just fine using http and not https

**Comment by MiniJennJenn** [comment]
Well the problem seems to be only in dota2abilities and dota2items which means it's not simply an https issue. For example, the cars trivia. Will look more into it though. All the other image based trivias don't seem to have any issues.

Awk: didn't refresh page and see palm's responses. oops

**Comment by sverit** [comment]
**tl;dr:** Remove the "h" right before the ".png" in the imgur links from the dota2abilities.txt and dota2items.txt and the image previews should work fine in discord. (Search/replace "h.png" with ".png").

**Explanation:**
The problem is actually a combination of the imgur links and discord, as it affects only those imgur links which do not work when the filetype extension is left out.

Example (from the dota trivia):
- https://i.imgur.com/ZUB9LCIh.png <br/>Direct link used in the trivia file. Works when opened in a browser as a link, but has no preview in discord. 
- https://i.imgur.com/ZUB9LCIh <br/>Removing only the filetype extension gives a 404 (this seems to be the reason for the missing preview in discord).
- https://i.imgur.com/ZUB9LCI and https://i.imgur.com/ZUB9LCI.png <br/>Removing the "h" at the end of the filename actually works (with filetype extension and without). This is the normal behaviour for imgur links.

Every imagelink in the two dota files has the smallcaps "h" at the end. I don't know why, but this small change should fix it. HTH.

**Comment by MiniJennJenn** [comment]
Oh interesting. Nice catch. I'll go patch that one in. Thanks a lot!

**Comment by Kowlin** [comment]
Fixed in #973, expecting merge. Closing this prematurely.
